views/alert_center.erb

```diff
@@ -0,0 +1,43 @@
+<h1>Manage Your Alert Groups</h1>
+
+<form method="POST">
+    <fieldset>
+        <div class="dcf-form-group">
+            <h4> General Alerts </h4>
+            <% generalAlerts.each do |alert| %>
+                <input <%= 'checked="checked"' if alert_ids.include?(alert.id)%> type="checkbox" name="alert_id_<%=alert.id%>" value=<%=alert.id%> /> <%= alert.name %> <br/>
+                <p>- <%=alert.description%> </p>
```

Comment:
Before showing the alert description for each of the alerts check to make sure the description isn't null or empty (could use .blank?) so that a blank line doesn't show up here on the alert center if there is no description. Example image of this here https://ctrl.vi/i/qQ0v9RBqu